src/metabase/sync/sync_metadata/fields.clj

```diff
@@ -131,7 +132,12 @@
       (log/info (format "Base type of %s has changed from '%s' to '%s'."
                         (field-metadata-name-for-logging table metabase-field)
                         old-base-type new-base-type))
-      (db/update! Field (u/get-id metabase-field), :base_type new-base-type))))
+      (db/update! Field (u/get-id metabase-field), :base_type new-base-type))
+    ;; And field comment, but only if the existing description is blank
+    (when (and (str/blank? old-field-comment) (not (str/blank? new-field-comment)))
+      (log/info (format "Comment has been added for %s."
+                        (field-metadata-name-for-logging table metabase-field)))
+      (db/update! Field (u/get-id metabase-field), :description new-field-comment))))
```

Comment:
this logic (checking to make sure `new-field-comment` isn't blank) is the sort of thing we need to have for Table as well